drivers/serial/Kconfig.it8xxx2

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2020 ITE Corporation. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig UART_ITE_IT8XXX2
+	bool "ITE serial uart driver"
+	select SERIAL_HAS_DRIVER
+	select SERIAL_SUPPORT_INTERRUPT
+	help
+	  This option enables the ITE serial uart driver.
+	  This driver can be used for the serial hardware
+	  available on ITE EV-boards.
+
+# ---------- Port 0 ----------
+
+config UART_ITE_IT8XXX2_PORT_0
```

Comment:
Shouldn't a hardware instance be enabled via DT?
@anangl

Reply:
see https://github.com/zephyrproject-rtos/zephyr/blob/master/drivers/serial/Kconfig.nrfx#L25

Reply:
I revise it as "def_bool $(dt_nodelabel_has_compat,uart1,it8xxx2-uart)"
The default config will depend on the device tree - uart device. 
(Bythe way, I'v also revised the interrupt config as this way. I am not sure should all the drivers config be enabled via dts?)
BR,